rosbag2_cpp/test/rosbag2_cpp/test_sequential_reader.cpp

```diff
@@ -210,3 +214,20 @@ TEST_F(SequentialReaderTest, next_file_calls_callback) {
   EXPECT_EQ(closed_file, bag_file_1_path_.string());
   EXPECT_EQ(opened_file, bag_file_2_path_.string());
 }
+
+TEST_F(TemporaryDirectoryFixture, reader_accepts_bare_file) {
+  const auto bag_path = rcpputils::fs::path(temporary_dir_path_) / "bag";
+  const auto expected_bagfile_path = bag_path / "bag_0.db3";
+  const auto expected_storage_implementation = "sqlite3";
+
+  {
+    // Create an empty bag with default storage
+    rosbag2_cpp::Writer writer;
+    writer.open(bag_path.string());
```

Comment:
The same comment in regards to the defualt storage.
> Perhaps it would be better to specify sqlite3 storage for writer explicitly since we check for it at the end of the tests.
In case if change default plugin to mcap this test is going to fail.

Reply:
I would rather specify no storage - so that if someday we don't ship sqlite3 by default, this test doesn't break. But, I agree that hardcoding the result breaks that future case anyways.